### ISSUE TITLE
Error: Undefined name 'Overflow' with Flutter Channel master, 2.13.0-0.0.pre.297

### ISSUE THREAD

**Issue by jmgeffroy** [peiffer-innovations/json_theme#36]
Hi, 
This is close to [another closed issue](https://github.com/peiffer-innovations/json_theme/issues/35). But I had this issue with my previous version of Flutter, then upgraded right now, and still getting this error. I see that [Overflow has been deprecated for a while](https://api.flutter.dev/flutter/rendering/Overflow.html); Maybe it's completely removed from these latest Flutter releases?

The error message is:

```
.pub-cache/hosted/pub.dartlang.org/json_theme-3.3.1/lib/src/codec/theme_encoder.dart:4082:14: Error: Undefined name 'Overflow'.
``` 

My doctor -v output:

```
[✓] Flutter (Channel master, 2.13.0-0.0.pre.297, on macOS 12.0.1 21A559 darwin-arm, locale fr-FR)
    • Flutter version 2.13.0-0.0.pre.297 at /Applications/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b4325b68a2 (il y a 4 heures), 2022-03-28 21:57:44 -0700
    • Engine revision e21d49ae26
    • Dart version 2.17.0 (build 2.17.0-246.0.dev)
    • DevTools version 2.11.4

[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at /Applications/android-sdk-macosx
    • Platform android-32, build-tools 32.0.0
    • ANDROID_HOME = /Applications/android-sdk-macosx
    • Java binary at: /Library/Java/JavaVirtualMachines/jdk-11.0.7.jdk/Contents/Home/bin/java
    • Java version Java(TM) SE Runtime Environment 18.9 (build 11.0.7+8-LTS)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode-13.2.1.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] Android Studio (version 2021.1)
    • Android Studio at /Users/jmgeffroy/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/211.7628.21.2111.8092744/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)

[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/jmgeffroy/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7784292/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] IntelliJ IDEA Ultimate Edition (version 2021.3.3)
    • IntelliJ at /Users/jmgeffroy/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin version 65.2.4
    • Dart plugin version 213.7227

[✓] IntelliJ IDEA Ultimate Edition (version 2021.3.3)
    • IntelliJ at /Users/jmgeffroy/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/213.7172.25/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] IntelliJ IDEA Ultimate Edition (version 2021.3.2)
    • IntelliJ at /Users/jmgeffroy/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/213.6777.52/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] Connected device (3 available)
    • SM N9750 (mobile) • RF8N21SXBWY • android-arm64  • Android 12 (API 31)
    • macOS (desktop)   • macos       • darwin-arm64   • macOS 12.0.1 21A559 darwin-arm
    • Chrome (web)      • chrome      • web-javascript • Google Chrome 99.0.4844.84

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by jmgeffroy** [comment]
I confirmed this enum has disappeared in a recent update in the master channel. Here is a diff of stack.dart between March 22 and today:

<img width="941" alt="Capture d’écran 2022-03-29 à 10 06 39" src="https://user-images.githubusercontent.com/4281439/160576422-55fa2a6a-0645-4b1a-8409-f238fdcbd518.png">

**Comment by jpeiffer** [comment]
Thanks for the report.  I tend not to follow the `beta` and `master` channels so it's always helpful when someone finds an issue in them and reports it.

I just published [3.3.1-pre113+1](https://pub.dev/packages/json_theme/versions/3.3.1-pre113+1) and that should fix your problem in `master`.

I'm going to go ahead and close the issue.  Feel free to reopen it if you don't believe it is resolved with that pre-release deployment.

**Comment by aideric** [comment]
Flutter just release 3.0 in the stable channel.
Type 'Overflow' not found happen in the latest Flutter 3.0.
If we use the latest master branch version of this library. 
There will be another error occur in automated_testing_framework.
```
automated_testing_framework-3.3.1+1/lib/src/widgets/testable_text_form_field.dart:181:11: Error: No named parameter with the name 'maxLengthEnforced'.
              maxLengthEnforced: widget.maxLengthEnforced,
```
Please release [3.3.1-pre113+1](https://pub.dev/packages/json_theme/versions/3.3.1-pre113+1) to us. Thank you.
